e2e/libs/volume/crd.py

```diff
@@ -65,11 +66,20 @@ def create(self, volume_name, size, numberOfReplicas, frontend, migratable, data
                 plural="volumes",
                 body=body
             )
-            if not Standby:
-                self.wait_for_volume_state(volume_name, "detached")
-            else:
+            if Standby:
                 self.wait_for_volume_state(volume_name, "attached")
-            self.wait_for_restore_required_status(volume_name, Standby)
+            else:
+                if fromBackup:
+                    if wait_attached:
+                        self.wait_for_volume_state(volume_name, "attached")
+                    else:
+                        self.wait_for_volume_state(volume_name, "detached")
+                else:
+                    self.wait_for_volume_state(volume_name, "detached")
+            if wait_attached:
```

Comment:
Why does whether to `wait_for_restore_required_status` depends on `wait_attached`? Should it depend on if it's restored from a backup?

Reply:
When we create a volume from a backup, `restoreRequired` will be `true`. Once the restore is complete and the volume is `detached`, `restoreRequired` will become `false`. If we use `fromBackup` as the condition, it may cause failures in other existing cases that involve `creating volume from backup`.

Reply:
Could I know which existing cases will fail?

Reply:
> Why does whether to `wait_for_restore_required_status` depends on `wait_attached`? Should it depend on if it's restored from a backup?

Hi @yangchiu 
    I understand your point, and I will update it.